apps/core/src/hooks/useGetDynamicFieldObject.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '../api/RpcClientContext';
+import { useQuery } from '@tanstack/react-query';
+
+type DynamicFieldName = {
+    type: string;
+    value?: string;
+};
+
+export function useGetDynamicFieldObject(
+    parentId: string,
+    name: DynamicFieldName
+) {
+    const rpc = useRpcClient();
+    return useQuery(
+        ['dynamic-fields-object', parentId, name],
+        () =>
+            rpc.getDynamicFieldObject({
```

Comment:
It seems that this API always fails with errors like `Cannot find dynamic field [DynamicFieldName...` do you know why? I've tested it locally with a Hero object.

https://user-images.githubusercontent.com/10210143/231285539-6f295370-893b-4bd9-b6de-336a213e1781.mov

Reply:
I think `suix_getDynamicFieldObject` fails  can you check